control/utils.py

```diff
@@ -244,7 +244,7 @@ def __init__(self, config=None):
                 max_log_file_size = 0
                 max_log_files_count = 0
             try:
-                os.makedirs(self.log_directory, 0o777, True)
+                os.makedirs(self.log_directory, 0o755, True)
```

Comment:
Unfortunately, we got a lot of issues trying to do things externally because of the different build environments. Things which worked in upstream didn't work in downstream. Anyway, this is old code. I just changed the mode so it wouldn't be writable by other user ids.